ansible/roles/containerd/tasks/main.yaml

```diff
@@ -4,6 +4,12 @@
     path: "/etc/systemd/system/containerd.service.d/"
     state: directory
 
+- name: Create containerd max files
+  template:
+    dest: /etc/systemd/system/containerd.service.d/max-files.conf
+    src: etc/systemd/system/containerd.service.d/max-files.conf
+    mode: 0644
```

Comment:
@faiq I started to rebase my molecule test PR #553 

it turns out that this part fails with Ubuntu. This is when `http_proxy` or `https_proxy`is not set and the first task is not run due to that. 

My suggestion:

Move this part after install includes and remove the conditions from directory creation

Reply:
@faiq cherry-picking 137be77639a90cb75cdc3d55831404e02b85823a should help